attribution_scopes.md

```diff
@@ -31,38 +31,40 @@ In general, the approach here is to allow API callers to specify a list of strin
 
 ### API changes
 
-The following optional parameters will be added to the JSON in `Attribution-Reporting-Register-Source` during source registration: `attribution_scopes`, `attribution_scope_limit`, and `max_event_states`:
+The following optional parameters will be added to the JSON in `Attribution-Reporting-Register-Source` during source registration: `attribution_scopes`, `values`, `limit`, and `max_event_states`:
```

Comment:
I would phrase this a bit differently, as `values` and `limit` are actually required within an `attribution_scopes` dictionary.